src/ynEIGEN/WithdrawalsProcessor.sol

```diff
@@ -157,13 +166,17 @@ contract WithdrawalsProcessor is IWithdrawalsProcessor, Initializable, AccessCon
     }
 
     /// @notice Gets the total pending withdrawal requests
-    /// @return _pendingWithdrawalRequests The total pending withdrawal requests
-    function getPendingWithdrawalRequests() public view returns (uint256 _pendingWithdrawalRequests) {
-        uint256 _pendingUnqueudRequests = withdrawalQueueManager.pendingRequestedRedemptionAmount() - totalQueuedWithdrawals;
-        uint256 _availableRedemptionAssets = redemptionAssetsVault.availableRedemptionAssets();
-        if (_availableRedemptionAssets >= _pendingUnqueudRequests) revert CurrentRedemptionAssetsSufficient();
-        _pendingWithdrawalRequests = _pendingUnqueudRequests - _availableRedemptionAssets;
-        if (_pendingWithdrawalRequests <= minPendingWithdrawalRequestAmount) revert PendingWithdrawalRequestsTooLow();
+    /// @return deficitAmount The total pending withdrawal requests
+    function getPendingWithdrawalRequests() public view returns (uint256 deficitAmount) {
+        uint256 pendingAmount = withdrawalQueueManager.pendingRequestedRedemptionAmount();
+        uint256 availableAmount = redemptionAssetsVault.availableRedemptionAssets() + totalQueuedWithdrawals;
+        if (pendingAmount <= availableAmount) {
```

Comment:
shouldn't this be `pendingAmount < availableAmount` (as oppposed to <=)

Reply:
this may be semantically correct but the condition name does not match what the mathematical statement says

CurrentAvailableAmountIsSufficient

Available amount is insufficient - means there's to little available amount

the if is "pendingAmount <= availableAmount" these 2 statements are contradictory